plugins/CuraEngineBackend/StartSliceJob.py

```diff
@@ -340,6 +340,11 @@ def run(self) -> None:
         self._slice_message.sentry_id = f"{user_id}"
         self._slice_message.cura_version = CuraVersion
 
+        # Add the project name to the message if the user allows for non-anonymous crash data collection.
+        account = CuraApplication.getInstance().getCuraAPI().account
+        if account and account.isLoggedIn and not CuraApplication.getInstance().getPreferences().getValue("info/anonymous_engine_crash_report"):
```

Comment:
Maybe we don't need the user to be logged in to get the crash report, he has already acknowledged sending non-anonymous data and that's goot enough

```suggestion
        if not CuraApplication.getInstance().getPreferences().getValue("info/anonymous_engine_crash_report"):
```

Reply:
Then we need to store the User Name in the config since that won't be accessible if not logged in.

But the better reason to not do that is that it is not guaranteed that the user who logged out is still the same as who is using Cura at this moment. There is after all a reason why he logged out.

Reply:
We can still send the project name if the user is not logged in. I do not mean to send the use name in this case, but at least the project name gives us some information.

Reply:
I prefer to leave it as is